sql/parser/sql.y

```diff
@@ -1575,8 +1575,7 @@ createdb_opt_item:
 // option will silently break. Best defense is to provide a regression test
 // exercising every such option, at least at the syntax level.
 createdb_opt_name:
-  IDENT {}
-| CONNECTION LIMIT {}
+  CONNECTION LIMIT {}
```

Comment:
Options of "CREATE DATABASE" are supposed in many popular DB  in the market, maybe it's necessary to reserve for user to substitute esaily in the future.

Cocoroach has no plan to implement it currently?  even later?